plugins/src/webrtcsink/imp.rs

```diff
@@ -1749,29 +1744,28 @@ impl WebRTCSink {
         let mut state = element.imp().state.lock().unwrap();
 
         if let Some(consumer) = state.consumers.get_mut(peer_id) {
+            // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
+            // that it is not worth it below that threshold
             let fec_ratio = {
-                // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
-                // that it is not worth it below that threshold
-                if bitrate <= 2_000_000 || consumer.cc_info.max_bitrate <= 2_000_000 {
+                if bitrate <= DO_FEC_THRESHOLD || consumer.cc_info.max_bitrate <= DO_FEC_THRESHOLD {
                     0f64
                 } else {
-                    (bitrate as f64 - 2_000_000.)
-                        / (consumer.cc_info.max_bitrate as f64 - 2_000_000.)
+                    (bitrate as f64 - DO_FEC_THRESHOLD as f64)
+                        / (consumer.cc_info.max_bitrate as f64 - DO_FEC_THRESHOLD as f64)
                 }
             };
 
-            let fec_percentage = fec_ratio * 50f64;
-            let encoders_bitrate = ((bitrate as f64) / (1. + (fec_percentage / 100.))) as i32;
+            let fec_percentage = (fec_ratio * 50f64) as u32;
```

Comment:
My understanding is that `bitrate` is the full bitrate for a particular stream, including the FEC percentage, in practice this means that we need to set the `encoder_bitrate` as bitrate / fec_percentage as it was the case before.'

Why do you think it is not the case anymore?

(we should add unit tests :D)

Reply:
I don't :) 
You're right, fixed that. Thank you!